style.css

```diff
@@ -16,15 +16,9 @@ a {
 }
 
 h1 {
-    display: inline-block;
-    margin: 0;
     font-size: 2.125rem;
 }
 
-p {
-    font-size: 1rem;
-}
-
 @media screen and (min-height: 375px) {
```

Comment:
🟡 Out of scope for this PR, just general form improvement for the future:

Having `@media screen and (min-height: 375px)` & `@media screen and not (min-height: 375px)` means one of them is redundant - this is like

```
if condition
else if !condition
```

instead of 

```
if condition
else
```

Not a blocker, just a weird engineering choice that adds unnecessary complexity

Reply:
Would suggest writing your CSS with the following assumptions:

1. Light theme is the default
2. Small screens are the default

(this is the general standard used by tailwind, bootstrap, most frameworks).

Then your CSS for light & small screens is the baseline, and you override specific bits as you go

ie, pseudocode

```
text colour: black;
padding: 0;

// Defaults for large screens
@media screen and (min-width:: 100px) {
  padding: 1rem 16rem;
}

// Defaults for dark screens
@media (prefers-color-scheme: dark) {
  text colour: white;
}
```

Reply:
(which reminds me, you could benefit from reducing the horizontal padding on narrow screens)

![CleanShot 2024-01-31 at 19 17 00@2x](https://github.com/ReeceDrage/personal-website/assets/45462299/8a0c4a15-c148-46e0-b4d5-cb26c21cb6e6)